README.md

```diff
@@ -1700,6 +1700,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Shadman Ali](https://github.com/shadmanhere)
   - [Shafaq Israil](https://github.com/shafaq-here)
   - [Shahan](https://github.com/shahan007)
+    [Shahank Dwivedi](https://github.com/Imshk)
```

Comment:
Missing a colon.
```suggestion
  - [Shahank Dwivedi](https://github.com/Imshk)
```

Reply:
Both seems same, I Don't think you have made any changes .

Reply:
Yes, I made a change (see the `-`).